noir-projects/aztec-nr/aztec/src/messages/logs/note.nr

```diff
@@ -33,51 +33,75 @@ pub fn compute_note_log<Note>(
 where
     Note: NoteType + Packable,
 {
-    compute_log(note, storage_slot, recipient, PRIVATE_NOTE_MSG_TYPE_ID)
+    let message_plaintext = private_note_to_message_plaintext(note, storage_slot);
+    let message_ciphertext = AES128::encrypt_log(message_plaintext, recipient);
+
+    prefix_with_tag(message_ciphertext, recipient)
 }
 
-pub fn compute_partial_note_log<Note>(
-    note: Note,
+pub fn compute_partial_note_private_content_log<PartialNotePrivateContent>(
```

Comment:
I split these two because the fact that private note and partial note encoding matches is just a coincidence. This was not the case in the past, and so the merge made sense, but now that we have msg ids these are two separate concepts.